lib/hanami/model.rb

```diff
@@ -73,6 +73,7 @@ def self.container
 
     # @since 0.1.0
     def self.load!(&blk) # rubocop:disable Metrics/AbcSize
+      configuration.gateway.use_logger(configuration.logger)       unless configuration.logger.nil?
```

Comment:
What do you think of introducing a `NullObject`/`Mimic` logger? With ti we would not check for nils and it would simply do nothing. Yeah, one more method call, but cleaner code.